.github/workflows/reusable-build-wheel.yml

```diff
@@ -75,8 +81,10 @@ jobs:
     - name: Upload built artifacts for testing and publishing
       uses: actions/upload-artifact@v4
       with:
-        name: >-
-          ${{ inputs.dists-artifact-name }}-${{ inputs.os }}-${{ inputs.qemu }}
+        name: ${{ inputs.dists-artifact-name }}-
+           ${{ inputs.os }}-
+           ${{ inputs.qemu }}-
+           ${{ inputs.tag }}
```

Comment:
@bdraco I recently discovered that this would break because of asterisks: https://github.com/ansible/pylibssh/pull/676#issuecomment-2656574936.

And so I ended up with a solution where I compute a hash of all the inputs: [ansible/pylibssh@`90e86aa` (#676)](https://github.com/ansible/pylibssh/pull/676/commits/90e86aa39174723effb05bf151360d9448880f84#diff-3826dffee70012c1e07ee3d63a4cf5ba7ed82c17bbac68145432a97e71dbade7R95-R112).
Then, I append it: [ansible/pylibssh@`90e86aa` (#676)](https://github.com/ansible/pylibssh/pull/676/commits/90e86aa39174723effb05bf151360d9448880f84#diff-3826dffee70012c1e07ee3d63a4cf5ba7ed82c17bbac68145432a97e71dbade7R213).

Plz migrate that solution here instead.

Reply:
Thanks. Will adjust when time permits.

Reply:
migrated in 656ee41af45ce42929346360b9aa78d6615a9f42